server/src/routes/api/v1/voteRouter.js

```diff
@@ -0,0 +1,44 @@
+import express from "express";
+import { Review, Vote } from "../../../models/index.js";
+import { ValidationError } from "objection";
+import ReviewSerializer from "../../../serializers/ReviewSerializer.js"
+
+const voteRouter = new express.Router();
+
+voteRouter.post("/", async (req, res) => {
+    const voteValue = req.body.voteValue;
+    const reviewId = req.body.reviewId;
+    const userId = req.user.id;
+    try {
+        const voteExists = await Vote.query().where("reviewId",`${reviewId}`).where("userId",`${userId}`)
```

Comment:
replace this with Vote.query().findOne(), since a user should only ever have one vote for one review
also string interpolation shouldnt be needed